dispatch/dispatch.go

```diff
@@ -81,6 +81,7 @@ type AlertBlock struct {
 type APIAlert struct {
 	*model.Alert
 	Status types.AlertStatus `json:"status"`
+	ID     string            `json:"id"`
```

Comment:
I am not sure if fingerprints are entirely unique, so thereby `id` might be misleading. @fabxc What do you think?